src/bazel/tasks.ts

```diff
@@ -56,9 +56,7 @@ export function createBazelTask(
   const targetsDescription = options.targets.join(", ");
   const task = new vscode.Task(
     { type: "bazel", command, targets: options.targets },
-    // TODO(allevato): Change Workspace to Global once the fix for
-    // Microsoft/vscode#63951 is in a stable release.
-    options.workspaceInfo.workspaceFolder || vscode.TaskScope.Workspace,
```

Comment:
There's a subtle difference here that's being lost. If the command options referred to a workspace folder, the created task would be scoped to that workspace folder; _otherwise_ the task would be created at `Workspace` scope (which was a substitute for `Global` scope due to that bug).

This change makes _all_ tasks globally scoped, which could cause different behaviors depending on what other configurations those workspace folders might have compared to the global (user) configuration.

In order to preserve the old behavior, you'll probably need to map the file system path of the `workspaceInfo` to the `vscode.workspace.WorkspaceFolder` value it corresponds to, since you no longer carry that information directly.

Reply:
Ah that makes more sense. I've updated the change so that tasks can accept the VSCode workspaceFolder and set the scope accordingly. I also pass that workspace folder whenever I create a task. PTAL.

Reply:
Please don't approve yet. There's more work to be done since there won't always be an active document.

Reply:
Sounds good, let me know when it's ready.